src/main/java/org/openrewrite/java/testing/jmockit/JMockitBlockRewriter.java

```diff
@@ -211,9 +219,11 @@ private void rewriteVerify(J.MethodInvocation invocation, @Nullable Expression t
 
         // do this last making sure rewrite worked and specify hasReference=false because in verify case it cannot find
         // the static reference in AddImport class, and getSelect() returns not null
-        visitor.maybeAddImport(MOCKITO_IMPORT_FQN_PREFX, "verify", false);
-        if (verificationMode != null) {
-            visitor.maybeAddImport(MOCKITO_IMPORT_FQN_PREFX, verificationMode);
+        if (!rewriteFailed) {
```

Comment:
I don't think this check is necessary, as `maybeAddImport` will only add the import if the type is actually used.

Reply:
Is an expensive check, especially if not needed, hence would like to skip it if not needed

Reply:
Fair enough, I wasn't aware of the expensive nature of `maybeAddImport`, but we also already know that the rewrite failed at this point so I see no harm in this check.